honeybee_energy/ventcool/crack.py

```diff
@@ -0,0 +1,165 @@
+# coding=utf-8
+"""Definition of surface crack for multizone airflow."""
+from __future__ import division
+
+from honeybee._lockable import lockable
+
+
+@lockable
+class AFNCrack(object):
+    """Definition of surface crack for multizone airflow.
+
+    Args:
+        air_mass_flow_coefficient_reference: A number in kg/s-m at 1 Pa per meter of
+            crack length at the conditions defined in the ReferenceCrack condition;
+            required to run an AirflowNetwork simulation. Some common values for this
+            coefficient from the DesignBuilder Cracks template include the following:
+
+                * 0.00001 - Tight low-leakage external wall
+                * 0.001 - Tight, low-leakage internal wall
+                * 0.0004 - Poor, high-leakage external wall
+                * 0.019 - Poor, high-leakage internal wall
+
+        air_mass_flow_exponent: An optional dimensionless number between 0.5 and 1 used
+            to calculate the crack mass flow rate; required to run an AirflowNetwork
+            simulation. This value represents the leak geometry impact on airflow, with
+            0.5 generally corresponding to turbulent orifice flow and 1 generally
+            corresponding to laminar flow. The default of 0.65 is representative of many
+            cases of wall and window leakage, used when the exponent cannot be measured.
+            (Default: 0.65).
+        crack_factor: A number indicating multiplier for air mass flow through a crack.
+            (Default: 1).
+
+    Properties:
+        * air_mass_flow_coefficient_reference
+        * air_mass_flow_exponent
```

Comment:
These are really long variable names. Can we just call them `flow_coefficient` and `flow_exponent`?  Generally speaking, it gets hard to write code with variables that are longer than 25 characters since these take up 1/3 of the whole character limit in a line.

Reply:
Yeah I agree, I just suck at variable naming. Will change.